app/services/announcements.ts

```diff
@@ -45,8 +47,24 @@ export class AnnouncementsService extends StatefulService<IAnnouncementsInfo> {
     await this.postBannerClose();
   }
 
+  get activationTimestamp(): number {
+    if (localStorage.getItem(this.localStorageKey)) {
+      return parseInt(localStorage.getItem(this.localStorageKey), 10);
+    }
+    this.activationTimestamp = Date.now();
+    return this.activationTimestamp;
+  }
+
+  set activationTimestamp(timestamp: number) {
+    localStorage.setItem(this.localStorageKey, timestamp.toString());
+  }
+
+  underOneWeek() {
+    return (Date.now() - this.activationTimestamp) / (1000 * 60 * 60 * 24 * 7) < 1;
```

Comment:
Why not just
```TypeScript
return (Date.now() - this.activationTimestamp) < (1000 * 60 * 60 * 24 * 7);
````
Doesn't really matter, just curious why the extra division before the comparison.

Reply:
Hmm good point. Will update.